flake.nix

```diff
@@ -0,0 +1,68 @@
+{
+  description = "Ontime is a browser-based application that manages event rundowns, scheduling, and cueing.";
+
+  inputs = {
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  };
+
+  outputs =
+    inputs@{ flake-parts, ... }:
+    flake-parts.lib.mkFlake { inherit inputs; } {
+      systems = [
```

Comment:
You could use https://github.com/nix-systems so the systems can be overidden by consumers of the flake.